docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -13,21 +13,65 @@ components: FilledInput, FormControl, FormHelperText, Input, InputAdornment, Inp
 
 The `TextField` wrapper component is a complete form control including a label, input and help text.
 
-{{"demo": "pages/components/text-fields/TextFields.js"}}
+`TextField` supports standard, outlined and filled styling.
```

Comment:
General concern regarding the usage of ``. It seems to break the flow of reading. Should we reduce its usage or change the styles to be less "distracting"?

Reply:
Not for this PR, but I would make the same case for the overuse of **bold text**.